server/controllers/tasks/lib/tasks.ts

```diff
@@ -108,7 +119,7 @@ function getKeys (uris: EntityUri[], includeArchived?: boolean) {
 
 const buildKey = state => uri => [ uri, state ]
 
-function completeWithEmptyArrays (getTasksByUris, uris: EntityUri[]) {
+const fillWithEmptyArrays = (getTasksByUris, uris: EntityUri[]) => {
```

Comment:
[non-blocking]
The stylistic trend I follow is rather transitioning the other way around (from arrow functions to named functions), outside of callbacks

Reply:
fixed in 8c45d0cab